### PR TITLE
Added docker-machine version 0.1.0

### PR DESCRIPTION
**Pull request by vlesierse**
Added docker-machine version 0.1.0 like docker-swarm this release no longer has a RC tag.

### CODE REVIEW

File: Library/Formula/docker-machine.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  go_resource "golang.org/x/tools" do
+    url "https://github.com/golang/tools.git", :revision => "473fd854f8276c0b22f17fb458aa8f1a0e2cf5f5"
+  end
+
+  go_resource "github.com/docker/machine" do
+    url "https://github.com/docker/machine.git", :revision => "eaff7635fc821d783eb6e996dbdad22dacdd058c"
+  end
+
+  def install
+  	ENV["GOPATH"] = buildpath
```
Comment (DomT4) [inline Library/Formula/docker-machine.rb]: Indentation is off here.

File: Library/Formula/docker-machine.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd "src/github.com/tools/godep" do
+      system "go", "install"
+    end
+
+    system "./bin/godep", "go", "build", "-o", "docker-machine", "."
+  	bin.install "docker-machine"
```
Comment (DomT4) [inline Library/Formula/docker-machine.rb]: And here too.

File: Library/Formula/docker-machine.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  end
+
+  test do
+  	output = shell_output("#{bin}/docker-machine --version")
```
Comment (DomT4) [inline Library/Formula/docker-machine.rb]: Here as well.

**Comment by vlesierse** [comment]
Done, Fixed the formatting.

**Comment by DomT4** [comment]
Thanks!

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Comment by DomT4** [comment]
As a general thing which I forgot to mention before, when submitting formulae to Homebrew/homebrew that already exist in another Homebrew tap, such as ` devel-only ` in this case, can people submit PRs to remove that formula from that tap?

That way I can get some heads up the maintainers here are okay with it and merge the PR to remove the tap version before something like this gets merged. Otherwise people run into when they update:

```
Warning: Could not create link for homebrew/devel-only/docker-machine, as it
conflicts with Homebrew/homebrew/docker-machine. You will need to use the
fully-qualified name when referring this formula, e.g.
  brew install homebrew/devel-only/docker-machine
```

Thanks!

**Comment by vlesierse** [comment]
Ah, ok thnx. I've got it noted.

**Pull request closed without merging**